releasenotes/notes/fix-elasticsearch-always-populate-span-tags-c2a212ec706b3f4b.yaml

```diff
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    elasticsearch: this fix resolves an issue where span tags were not fully populated on "sampled" spans, causing metric dimensions to be incorrect when spans were prematurely marked as sampled.
```

Comment:
Do the metric dimensions here refer to resource names?

Reply:
This particular customer cares about resource names, but generally there are several fields which were not calculated

Reply:
Makes sense, the new edit looks good to me too.